src/main/resources/bamm/meta-model/2.0.0/aspect-meta-model-shapes.ttl

```diff
@@ -930,7 +975,7 @@ bamm:ConstrainListTypeUsageShape
       sh:path mmm:listType ;
       sh:sparql [
          a sh:SPARQLConstraint ;
-         sh:message "mmm:listType may only be used for List Shapes" ;
+         sh:message "'{this}': mmm:listType may only be used for List Shapes" ;
```

Comment:
I have the feeling that this error message is not written in the right way. The term "Shape" is only on the level of how BAMM is specified ("meta meta model"), not for users that use BAMM, am I right?